### PR TITLE
Update/agenda routing

### PR DESCRIPTION
**Pull request by scott062**
Update GET single vs GET many agendas to use common endpoint (no need for nested calls in mapping)

https://github.com/MetaPhase-Consulting/mock-fsbid/pull/276

### CODE REVIEW

File: talentmap_api/fsbid/views/agenda.py [hunk 1]
```diff
@@ -16,7 +16,24 @@
 
 
 class AgendaItemView(BaseView):
-    permission_classes = [Or(isDjangoGroupMember('cdo'), isDjangoGroupMember('bureau'), isDjangoGroupMember('ao_user'),)]
+    permission_classes = [Or(isDjangoGroupMember('cdo'), isDjangoGroupMember('ao_user'),)]
```
Comment (mjoyce91) [inline talentmap_api/fsbid/views/agenda.py:19]: Trying to think - if we used this same endpoint to display assignment history on a public profile, should Bureau/Post users also be able to access?

File: talentmap_api/fsbid/views/agenda.py [hunk 2]
```diff
@@ -16,7 +16,24 @@
...
+        ])
+
+    def get(self, request, pk):
+        '''
+        Get single agenda by ai_seq_num
+        '''
+        return Response(services.get_single_agenda_item(request.META['HTTP_JWT'], pk))
+
+class AgendaItemListView(BaseView):
+    permission_classes = [Or(isDjangoGroupMember('cdo'), isDjangoGroupMember('ao_user'),)]
```
Comment (mjoyce91) [inline talentmap_api/fsbid/views/agenda.py:36]: Same question as https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/672/files#r831513964

File: talentmap_api/fsbid/views/agenda.py (hunk 1 above)
Comment (scott062) [reply talentmap_api/fsbid/views/agenda.py:19]: Shouldn't for agenda item view - maybe for a separate assignments view

**Comment by scott062** [comment]
TO-DO: Update or check FE mappings (noticed some missing data on UI)

File: talentmap_api/fsbid/views/agenda.py (hunk 1 above)
Comment (mjoyce91) [reply talentmap_api/fsbid/views/agenda.py:19]: Ok somehow I misread this PR as "assignment" instead of "agenda item" so that makes way more sense now

**Comment by mjoyce91** [comment]
> TO-DO: Update or check FE mappings (noticed some missing data on UI)

@scott062 - does that mean we should add a WIP label to this PR for the time being?

File: talentmap_api/fsbid/views/agenda.py [hunk 3]
```diff
@@ -16,7 +16,24 @@
 
 
 class AgendaItemView(BaseView):
-    permission_classes = [Or(isDjangoGroupMember('cdo'), isDjangoGroupMember('bureau'), isDjangoGroupMember('ao_user'),)]
+    permission_classes = [Or(isDjangoGroupMember('cdo'), isDjangoGroupMember('ao_user'),)]
+
+    @swagger_auto_schema(
+        manual_parameters=[
+            openapi.Parameter("perdet", openapi.IN_QUERY, type=openapi.TYPE_STRING, description='Perdet of the employee'),
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/views/agenda.py:23]: image
since id is part of the url, swagger auto generates an input for it

**Comment by elizabeth-jimenez** [comment]
`response = fetch_method(f"{uri}{id}" if use_id else uri, {"id": id} if use_id else {}, query_mapping_function, jwt_token, mapping_function, api_root)
` just gotta flip the use_id logic in `get_individual` in `common.py` i'll check if any other EPs use this ()

**Comment by scott062** [comment]
Merging now and opening new PR to address this stuff - Don't want to muddy what's going in 8.1.1

**Pull request merged**